gslib/utils/text_util.py

```diff
@@ -263,99 +263,78 @@ def PrintableStr(input_val):
   """
   return input_val
 
+def print_to_fd(*objects, **kwargs):
+    """A Python 2/3 compatible analogue to the print function.
+
+    This function writes text to a file descriptor as the
+    builtin print function would, favoring unicode encoding.
+
+    Aguments and return values are the same as documented in
+    the Python 2 print function.
+    """
+    def _get_args(**kwargs):
```

Comment:
Good catch, will fix spacing :+1: I kind of like moving `_get_args()` into its own private method since its logic is auxiliary to the function's purpose, but I'm happy to remove this function and just put it all together if that's the preferred way for everyone